subsquid/scripts/init.sh

```diff
@@ -1,7 +1,44 @@
 #!/bin/sh
 
+set -eou pipefail
+
+# main entry point for the script
+main() {
+  #  ${foo:="default value if unset"}
+  if ${TRIGGER_REPROCESSING:=false} ; then
+    run_steps_to_trigger_reprocessing
+    export TRIGGER_REPROCESSING=false
+  fi
+
+  log INFO "Running db migration"
+  npx squid-typeorm-migration apply || log DIE "DB migration step failed"
+
+  log INFO "Running sqlinit "
+  node lib/helperInit.js || log DIE  "Sqlinit step failed"
+
+  log INFO "Starting the processor" 
+  node -r dotenv/config lib/processor.js || log DIE "Starting the processor step failed"
+}
+
+run_steps_to_trigger_reprocessing() {
+  log INFO "Reprocessing triggered"
+  set_db_connection_details
+  psql -c "DROP SCHEMA public CASCADE ;
+           DROP SCHEMA squid_processor cascade ;
+           create SCHEMA public  ;
+           create schema squid_processor ;"
```

Comment:
Not sure if required but are we missing the grants?

Reply:
I tested on stating and the grants are not needed ; for prod I will need add them, but it's just a one time thing because the `postgres` user was used